ContactManagerApp/Pages/Index.cshtml.cs

```diff
@@ -48,7 +54,6 @@ public async Task OnGetAsync(CancellationToken cancellationToken)
         {
             RedirectToPage("localhost:7296");
```

Comment:
`RedirectToPage("localhost:7296");`

so sache ghöred eig nie in code. 

warschinlich wür 

`RedirectToPage("/");` 
au funktioniere